TAO/TAO_IDL/docs/annotations.md

```diff
@@ -74,16 +78,42 @@ struct Report {
 };
 ```
 
-### Special Cases of Annotations
+## What Can Be Annotated
 
 Annotations "may be applied to any IDL constructs or sub-constructs", as
 defined by the OMG. This is very vague and the OMG has not clarified this as of
-IDL 4.2. [(Also see Limitation #1)](#limitation1). What can be said about it
-though is that other than before normal declarations, like before structs,
-typedefs, and constants, in TAO\_IDL as of writing, annotations can also be
-applied in the following cases. See ["Reading Special Cases of
-Annotations"](#reading-special-cases-of-annotations) for how to have the
-compiler use these kinds of annotations.
+IDL 4.2 [(Limitation #1)](#limitation1). Because of the lack of standardization
+of what can be annotated and how, annotations for specific IDL elements have to
+be added on a as-needed basis.
+
+As of writing you can put annotations before the following things in IDL and a
+backend using TAO IDL should be able to read them:
```

Comment:
```suggestion
backend using TAO IDL's front-end library can use them:
```

Reply:
Between these two changes, this sentence is a lot more confident then mine.